### PR TITLE
How to Implement MLOps 

### LINKED ISSUE #639: How to Implement MLOps
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.
The use of machine learning and AI-based models in iOS,  chatbots, Stock price prediction, facial recognition, and detection, among other areas, has increased. Implementing all these technologies developed by combining AI and Machine Learning at scale is becomes challenging, and this is where MLOps comes in. MLOps is a set of practices that combine Machine Learning, DevOps, and Data Engineering to deploy and maintain ML systems in production reliability and efficiency. 
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
1. What is MLOps
2. Why MLOps matter
3. How to implement MLOps (with examples) 
### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #639

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/how-to-implement-mlops/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+MLOps includes a data science team for curating datasets and building AI models that analyze the datasets. Machine learning engineers are also part of the MLOps team, and they have the role of running datasets through the models in automated ways.
+
+There exist three levels of MLOps, as [described](https://cloud.google.com/solutions/machine-learning/mlops-continuous-delivery-and-automation-pipelines-in-machine-learning) by Google: MLOps level 0, MLOps level 1, and MLOps level 2. The choice between each level is dependent on an organization&#39;s size and the number of machine learning algorithms they target to run.
+
+- **MLOps level 0 -** This means thatan organization recognizes the need to adopt machine learning practices and have an in-house data scientist capable of deploying and building models. However, ML workflow in that entity remains mainly manual.
```
Comment (lalith1403) [inline articles/how-to-implement-mlops/index.md:17]: The last sentence isn't clear. ML workflow in that entity? what do you mean by that?

File: articles/how-to-implement-mlops/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- **MLOps level 1 -** This involves machine learning pipeline automation whereby the organization introduces a pipeline to enhance continuous training. Model and data validation is entirely automatic. A reduction in model performance or the introduction of new data triggers retraining. Solutions operating in a constantly changing environment better use this scenario to respond proactively to shifts in price rates, customer behavior, or other factors.
+
+- **MLOps level 2 –** With this level, [CI/CD pipeline](https://semaphoreci.com/blog/cicd-pipeline#:~:text=What%20is%20a%20CI%2FCD,and%20enable%20fast%20product%20iterations.) automates the deployment of machine learning models and elements of machine learning [training pipelines in production](https://www.altexsoft.com/blog/machine-learning-pipeline/). Tech-driven organizations can rely on this level for hourly or daily retraining of models, updating models in minutes, and simultaneous redeployment of these models on multiple servers. Such organizations require an end-to-end MLOps cycle for them to survive.
```
Comment (lalith1403) [inline articles/how-to-implement-mlops/index.md:21]: A gentle one-line introduction to CI/CD pipeline would be better for covering a broader audience.

File: articles/how-to-implement-mlops/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### How to implement MLOps
+
+#### Speech Recognition
+
+Speech recognition employs machine learning technology to convert speech signals into digital data. Recognition accuracy is achieved through the application of neural network algorithms in speech recognition software. Machine learning models adapt to changes in language and demonstrate a high level of accuracy.
+
+MLOps programs are applied in speech recognition applications. A speech recognition app identifies context such as emotion and tone based on how individuals speak. Eventually, people introduce new phrases, and the general style of how they talk changes. Due to such changes, the machine learning model will exhibit model decay.
+
+The guidelines of MLOps level 1 help in such a scenario. A system under MLOps level 1 is run on the device to monitor the speech recognition model&#39;s predictive performance. The system triggers an alert when performance falls below or approaches the threshold. This means that the responsible team must train a new model using fresh data and then deploy it to replace the old production model.
```
Comment (lalith1403) [inline articles/how-to-implement-mlops/index.md:35]: Nice example!

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Amazing article with interesting examples. A few minor changes and additional explanations and this should be ready to go.

**Comment by EKahuha** [comment]
@lalith1403 I have updated the article.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**